bin/gitea-push.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+. bin/common.sh
+[ "$CI" = 'true' ] && exit 1
```

Comment:
```suggestion
[ -n "$CI" ] && exit 1
```

Reply:
I ask this because we agreed to depend on this since our latest talk about dealing with base ENV vars